client/app/scripts/charts/node.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { connect } from 'react-redux';
-import classNames from 'classnames';
+import cx from 'classnames';
```

Comment:
Yup, old habits. `cx` was the import of the old react.addons.classset that was superceded by classnames.